### DISCUSSION TITLE
Hoe definiëren we de definition of done?

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#26] [Q&A] [answered]
Voor de start van het project hebben we een duidelijke defintion of done voor user stories en code nodig

**Comment by rubenvdlinde** [comment]
Zie ook -> https://github.com/orgs/Klantinteractie-Servicesysteem/discussions/36 voor dingen die we zinnig kunnen overnemen

**Comment by rubenvdlinde** [comment, marked as answer]
- De aanpassingen staan op de acceptatie omgeving (https://acc.opencatalogi.nl/) en voldoen aan alle acceptatiecriteria. 
- De validatie groep is akkoord met de wijziging zo als gepresenteerd op de acceptatie omgeving
- Code is gereviewd zijn door een andere developer. 
- Automatische en kwaliteit tests
Alle tests zijn uitgevoerd en geslaagd en zichtbaar (als badges) op de documentatie (hoofdpagina van de codebase).
- Functionele test
Indiener / Eigenaar  van de user story heeft de uitvoering op de acceptatie omgeving getest, en zijn opmerkingen of akkoord als comment bij de user story gezet. 
- Documentatie is getest
Indiener / Eigenaar  van de user story heeft de documentatie op de acceptatie doorlezen, en zijn opmerkingen of akkoord als comment bij de user story gezet. 
- Applicatie moet snel zijn
Als gebruiker wil ik dat een detail pagina onder 1 seconde (1000MS ) inlaad
 Als gebruiker wil ik dat een zoek overzicht pagina onder 1,5 seconde (1500ms) inlaad
Dit is exclusief elementen die gevuld worden uit derden systemen

**Reply by RonaldvCortenberghe** [reply]
akkoord
